cmd/executor/main.go

```diff
@@ -101,8 +101,34 @@ func readConfiguration(config interface{}) {
 	}
 }
 
+func startExecutor(conf executor.Config) error {
```

Comment:
Move this code to the `executor` package.

Reply:
This would create an import cycle. Unless the hook code stays here...